Sources/Sentry/SentrySdkPackage+Private.h

```diff
@@ -0,0 +1,11 @@
+#if __has_include(<Sentry/SentryDefines.h>)
+#    import <Sentry/SentrySdkPackage.h>
+#else
+#    import "SentrySdkPackage.h"
+#endif
+
+@interface SentrySdkPackage ()
```

Comment:
`m`: SentrySDKPackage is internal. Why did you have to add a `+Private.h` header?